hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/ozone/container/common/TestEndPoint.java

```diff
@@ -182,7 +182,7 @@ public void testTmpDirCleanup() throws Exception {
         serverAddress, 1000)) {
       DatanodeDetails datanodeDetails = randomDatanodeDetails();
       OzoneContainer ozoneContainer = new OzoneContainer(
-          datanodeDetails, conf, getContext(datanodeDetails), null);
+          datanodeDetails, conf, getContext(datanodeDetails), null, null);
```

Comment:
nit: if you leave the constructor with the shortened parameter list, use it here as well as in other tests ;)

Reply:
Updated.